### ISSUE TITLE
Creating DOM elements from JavaScript?

### ISSUE THREAD

**Issue by micros123** [projecttouch/projecttouch#10]
Now that I opened one issue here, I might as well bring something else up :-)

In quite a few places, document.createElement() is used to create DOM elements. I'd like to propose to keep this to a minimum.

When the view element is static, just add it to index.html and define the jQuery selector in the options.el of a View-element.
When the view element is dynamic, let's use the template rendering power of Backbone/Underscore instead of writing a lot of JavaScript.
If we stick to this, it will be a lot clearer what kind of html structure we are creating, which as a bonus makes writing the css easier.

**Comment by gskinner** [comment]
I agree on this, it's much easier to scan through the presentation layer if the elements are in the HTML. You can always tag elements with an "off" class (visibility:hidden, display:none).

In the rare occasion that createElement is needed for an on-screen element, it's generally good to create a div, and set innerHTML versus assembling large hierarchies via appendChild. This also lets you establish a standard for exposing this HTML (ex. you could put any injected html at the head of your js files).

**Comment by afewyards** [comment]
okay, agreed. let's convert it to this.
